src/core/control/tools/ImageHandler.cpp

```diff
@@ -96,3 +100,10 @@ auto ImageHandler::insertImage(GFile* file, double x, double y) -> bool {
 
     return true;
 }
+
+auto ImageHandler::insertImage(GFile* file, double x, double y) -> bool {
+    auto [succ, img, width, height] = ImageHandler::createImage(file, x, y);
+    if (!succ)
+        return false;
```

Comment:
Here and elsewhere: According to the [coding conventions](https://github.com/xournalpp/xournalpp/wiki/Coding-conventions) you should always use curly braces even if the block only has one line.

> Each if, for, while etc. are followed by curly braces, even if it only contains one line of code (because this is a common case for errors, e.g. search for "Apple goto fail bug")

Reply:
Alright. I must have read this (and usually this is my code style as well), but I've read code without the curly braces in `luapi_application.h` so often that I thought this would be the convention. What I mean by this is that another refactoring of `luapi_application.h` is needed to conform to the code conventions.